repos/runtime.proj

```diff
@@ -11,5 +11,12 @@
       <RepositoryReference Include="runtime-portable" Condition="'$(TargetOS)' == 'Linux' and '$(BuildPortableRuntime)' == 'true'" />
   </ItemGroup>
 
+    <!-- Additional Targets -->
+  <Target Name="InstallJustBuiltRuntime" AfterTargets="RemoveBuiltPackagesFromCache">
+    <!-- Install the runtime that was just built to be used by downstream repos, namely, aspnetcore -->
+    <Exec Command="tar -xvf $(SourceBuiltAssetsDir)dotnet-runtime-$(runtimeOutputPackageVersion)-$(TargetRid).tar.gz -C $(DOTNET_INSTALL_DIR)" />
```

Comment:
What happens on a rebuild? What happens in bootstrapping when we build N using N? This seems pretty difficult to get right. Maybe it doesn't need to block merging this PR (so we can keep working on prebuilts in parallel ASAP) but I think we should find a less overwritey way immediately.

Reply:
Filed https://github.com/dotnet/source-build/issues/1854 for this.